Create 2023-12-numfocus-update.md

tnabtaf opened the pull request:
### Description

This will be sent to NumFOCUS on Dec 13 for inclusion in their community newsletter.  Please review/update before noon, Dec 13, US East coast time.

tnabtaf committed:
Create 2023-12-numfocus-update.md

This will be sent to NumFOCUS on Dec 13 for inclusion in their community newsletter.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

xhochy commented:
One thing we could include is also the move to a new recipe format: https://github.com/conda-incubator/ceps/blob/main/cep-13.md This is going to be a significant change, but as it is also not yet available to the end-user, it might also be better to omit. I'm a bit split between "just news, not actionable" and showing that we are also improving with significant speed on the building side of conda packages.

jaimergp commented on workspace/2023/12/2023-12-numfocus-update.md:
```diff
@@ -0,0 +1,24 @@
+**Releases**
+
+* **`conda`** [23.11.0 release](https://conda.org/blog/2023-12-07-november-releases/#changes-in-conda-23110) includes 
+  * The speedy `conda-libmamba-solver` as the default dependency solver.  This and other improvements make `conda` run 4-5x faster.
```
This was technically part of the 23.10 release, although it looks like we are listing things that have happened recently in the project?

jaimergp replied:
```suggestion
  * The speedy `conda-libmamba-solver` as the default dependency solver.  This and other improvements like parallel downloads make `conda` run 4-5x faster.
```

tnabtaf replied:
@jaimergp We didn't advertise the 23.10 release through NumFOCUS.  My thought is to heap it in here.

jaimergp commented on workspace/2023/12/2023-12-numfocus-update.md:
```diff
@@ -0,0 +1,24 @@
+  * Support for plugins, enabling you to customize your conda installation.
+  * Package builders can now include desktop icons across all operating systems.
+  * The [conda documentation](https://docs.conda.io/en/latest/) has been reorganized and updated.
+* [**`pixi`**](https://pixi.sh/) a new open-source conda-compatible package and environment manager led by the folks at prefix.dev. Implemented in Rust, using the recently intoruduced Rattler library as a foundation.  See [the announcement](https://prefix.dev/blog/launching_pixi) for details.
```
```suggestion
* [**`pixi`**](https://pixi.sh/) a new open-source conda-compatible package and environment manager led by the folks at prefix.dev. Implemented in Rust, using the recently introduced Rattler library as a foundation.  See [the announcement](https://prefix.dev/blog/launching_pixi) for details.
```

jaimergp commented on workspace/2023/12/2023-12-numfocus-update.md:
```diff
@@ -0,0 +1,24 @@
+  * Package builders can now include desktop icons across all operating systems.
+  * The [conda documentation](https://docs.conda.io/en/latest/) has been reorganized and updated.
+* [**`pixi`**](https://pixi.sh/) a new open-source conda-compatible package and environment manager led by the folks at prefix.dev. Implemented in Rust, using the recently intoruduced Rattler library as a foundation.  See [the announcement](https://prefix.dev/blog/launching_pixi) for details.
+* **`conda-build`** now supports `emscripten-wasm32` and `wasi-wasm32` platforms.
+* **`conda-lock`**, **`mamba`**, **`rattler-build`**, **`grayskull`**, **`quetz`**, **`rattler`**, **`condax`** and many other projects had new releases too.
```
We could link to the repos here.

tnabtaf replied:
Will do.  It's tedious, but it makes this list much more useful.

jaimergp commented on workspace/2023/12/2023-12-numfocus-update.md:
```diff
@@ -0,0 +1,24 @@
+
+**Releases**
+
+* **`conda`** [23.11.0 release](https://conda.org/blog/2023-12-07-november-releases/#changes-in-conda-23110) includes 
```
This makes it sound like all these things happened in the same release. What about:

```suggestion
* **`conda`** [23.11.0](https://conda.org/blog/2023-12-07-november-releases/#changes-in-conda-23110) closes the 23.x release series. 2023 has brought many features to `conda`, including: 
```

tnabtaf commented:
@xhochy Thanks for the pointer. I will add this.

tnabtaf committed:
Apply suggestions from code review

All suggestions are from @jaimergp.

Co-authored-by: jaimergp <jaimergp@users.noreply.github.com>

tnabtaf committed:
Update 2023-12-numfocus-update.md

Implementing suggestions from @xhochy and @jaimergp.  Many thanks for the input.

tnabtaf commented:
This has been sent to Arliss @ NumFOCUS.  Can be merged or closed.
